certified-connectors/LatinShare SHP Permissions/apiDefinition.swagger.json

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
@NicolasValenzuela This connector swagger is not same as the one which was certified. If you are not intentionally changing the swagger, Can you please replace it with what you have submitted with us in the certification portal.

Reply:
@sriyen-msft  
How can I certify arrogance? Can you send me this file to compare?
So I can replace it if necessary.

Reply:
How can I certify swagger ? Can you send me this file to compare?
So I can replace it if necessary. Or maybe it would be best to certify the current file?

Reply:
There are lot of differences in the swagger you have uploaded in the portal and this PR. 
For Ex.: 
![image](https://user-images.githubusercontent.com/56276394/108151459-92c3c280-708b-11eb-88a0-6c026ff07db9.png)

Reply:
Thank you, for the comparison. 
I will need to update the portal swagger with this new version and do the corresponding tests. 

I will close the pull while the upgrade process is taking place. 
Thanks.